### PR TITLE
Sync with original template

### LINKED ISSUE #26: Automate release process (sync with template)
Just sync with the original template to resolve this issue https://github.com/fredrikhgrelland/vagrant-hashistack-template/pull/43

### LINKED ISSUE #27: Sync with template

### PR DESCRIPTION
**Pull request by zhenik**
## Closes/fixes/resolves issue(s)?
- closes #26 
- closes #27 

## What was added/changed/fixed?
- Docs update
- Pre & post bootstrap and startup
- Automate release

## Checklist (after created PR)
- [x] Added reviewers
- [x] Added assignee(s)
- [x] Added label(s)
- [x] Added to project
- [x] Added to milestone

### CODE REVIEW

File: dev/vagrant/bootstrap/consul/post/510-poststart-example.yml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+- name: "{{ service }} - poststart"
+  debug:
+    msg: This is the first user defined poststart task ( 510-poststart-example.yml )
```
Comment (claesgill) [inline dev/vagrant/bootstrap/consul/post/510-poststart-example.yml:3]: Examples in `dev/vagrant/bootstrap/consul/*` can be deleted, right?

File: getting_started_vagrantbox.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
+## The goal of this guide
```
Comment (claesgill) [inline getting_started_vagrantbox.md:1]: `getting_started_modules` and `getting_Started_vagrantbox` can be deleted

File: dev/vagrant/bootstrap/consul/post/510-poststart-example.yml (hunk 1 above)
Comment (zhenik) [reply dev/vagrant/bootstrap/consul/post/510-poststart-example.yml:3]: yepp

**Review by pdmthorsrud — APPROVED** [review]
LGTM

**Review by claesgill — APPROVED** [review]
I see that `format.sh` probably can be removed. However I'll approve and you can choose what you'll do.

**Comment by zhenik** [comment]
> I see that `format.sh` probably can be removed. However I'll approve and you can choose what you'll do.
> 
> Edit: Aslo see that `versions.tf` is not a think anymore in the template? Should we do something about this as well?

- I have removed template_example. 
- I will remove `versions.tf` from the root of the repository (I think it was trying to fix https://github.com/fredrikhgrelland/vagrant-hashistack-template/issues/14). I would recommend focusing on the issue first before doing such a try

**Comment by claesgill** [comment]
> > I see that `format.sh` probably can be removed. However I'll approve and you can choose what you'll do.
> > Edit: Aslo see that `versions.tf` is not a think anymore in the template? Should we do something about this as well?
> 
> * I have removed template_example.
> * I will remove `versions.tf` from the root of the repository (I think it was trying to fix [fredrikhgrelland/vagrant-hashistack-template#14](https://github.com/fredrikhgrelland/vagrant-hashistack-template/issues/14)). I would recommend focusing on the issue first before doing such a try

Okey :+1: But the `format.sh` file is in root not in template_example: https://github.com/fredrikhgrelland/terraform-nomad-hive/blob/sync/format.sh

**Review by claesgill — APPROVED** [review]

**Review by dangernil — APPROVED** [review]
LGTM

**Review by pdmthorsrud — APPROVED** [review]

**Review by claesgill — APPROVED** [review]

**Pull request merged**